homeassistant/components/media_player/onkyo.py

```diff
@@ -142,9 +142,11 @@ def mute_volume(self, mute):
         else:
             self._receiver.command('audio-muting off')
 
-    def turn_on(self):
+    def turn_on(self, **kwargs):
```

Comment:
This will not work. This is part of the ABC which is not aware of any kwargs. Better just remove it. It's fine to handle it purely in the service.